src/utils/validation.js

```diff
@@ -0,0 +1,100 @@
+const {
+  MOVING_COMMAND,
+  ERROR_MESSAGE,
+  GAME_COMMANDS,
+  GAME_COMMAND,
+} = require('./constants');
+
+/**
+ * @param {string} value
+ * @returns {boolean | ERROR}
+ */
+const validInteger = (value) => {
+  const regex = /^[0-9]+$/;
+  const number = parseInt(value, 10);
+
+  if (regex.test(number)) {
+    return true;
+  }
+
+  throw new Error(ERROR_MESSAGE.wrong_input);
+};
+
+/**
+ * @param {any[]} list
+ * @param {string} value 배열안에 있는지 검증할 값
+ * @returns
+ */
+const validIncludeValue = (list, value) => {
+  if (list.includes(value)) {
+    return true;
+  }
+
+  throw new Error(ERROR_MESSAGE.wrong_input);
```

Comment:
조건을 확인하여 에러를 던지는 로직이 validation 내에서 반복되는데 아래와 하나의 함수로 만들어 사용하면 어떨까요?

```js
const validate =(condition, errorMsg)=>{
 if(condition){
  return true;
 }
 throw new Error(errorMsg);
}
```

```suggestion
const condition = list.includes(value);
validate(condition, ERROR_MESSAGE.wrong_input);
```